src/model/FieldToViewMappings.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import SimpleView from './SimpleView';
+import { ProteinNames, ProteinNamesData } from './ProteinNames';
+import { GeneNames, GeneNamesData } from './GeneNames';
+import { Organism, OrganismData } from './Organism';
+
+const FieldToViewMappings: {
+  [index: string]: (row: any) => JSX.Element | undefined;
+} = {
+  accession: (row: { primaryAccession: string }) => (
+    <SimpleView
+      termValue={row.primaryAccession}
+      linkTo={`/uniprotkb/${row.primaryAccession}`}
+    />
+  ),
+  id: (row: { uniProtId: string }) => <SimpleView termValue={row.uniProtId} />,
+  protein_name: (data: ProteinNamesData) => {
```

Comment:
We are using `row` and `data` interchangeably in this function, could be easier to read if we pick only one.

Reply:
Good point, `data` it is!